apps/frontend/src/components/layout/player.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Heart,
+  ListMusic,
+  Mic2,
+  MoreHorizontal,
+  Play,
+  Repeat,
+  Shuffle,
+  SkipBack,
+  SkipForward,
+  Volume2,
+} from 'lucide-react';
+import Image from 'next/image';
+
+import { Button } from '@/components/ui/button';
+import { Slider } from '@/components/ui/slider';
+
+export function Player() {
```

Comment:
For me the padding on this is a bit off. On the left its quite big, on the right its almost none.

Reply:
I can't see what could be off with the padding. On the left there will be an image, maybe that you didn't see.

Reply:
Ohh the img is the thing is missed